### PR TITLE
feat: Add support comment in `create view` for MySQL and MariaDb

### PR DESCRIPTION
**Pull request by jxnu-liguobin**

### CODE REVIEW

File: src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt [hunk 1]
```diff
@@ -5705,13 +5705,44 @@ Analyze Analyze():
     }
 }
 
+ColumnDefinition ColumnDefinitionItem():
```
Comment (manticore-projects) [inline src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt:5708]: There should be a similar Production already, please try to re-use it.

File: src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt [hunk 2]
```diff
@@ -5705,13 +5705,44 @@ Analyze Analyze():
...
+}
+{
+	( item=RelObjectName() { columnWithComment.withColumnName(item); } ) 
+	[ tk=<K_COMMENT> tk2=<S_CHAR_LITERAL> { columnWithComment.addColumnSpecs(tk.image).addColumnSpecs(tk2.image); } ]
+	{
+		return columnWithComment;
+	}
+}
+
+List<ColumnDefinition> ColumnDefinitionItemList():
```
Comment (manticore-projects) [inline src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt:5723]: Please try to use an `ExpressionList<Column>` here.

**Review by manticore-projects — CHANGES_REQUESTED** [review]
Thank you for your contribution.
It looks great in general but I wonder if we can reuse the existing Production `ColumnDefinition` and use an `ExpressionList` for the collection.

**Comment by manticore-projects** [comment]
Thank you for your very good contribution, we do appreciate!

**Comment by manticore-projects** [comment]
Please would you still like to fix the Grammar regarding
```text
Warning: Choice conflict in [...] construct at line 1774, column 5.
         Expansion nested within construct and expansion following construct
         have common prefixes, one of which is: "COMMENT"
         Consider using a lookahead of 2 or more for nested expansion.
```

**Comment by jxnu-liguobin** [comment]
> COMMENT

ok

**Pull request merged**